TACC/Core-Styles
# fix(o-table-wrap): …scroll.extra → …scroll-extra

## Overview

Fix bugs with table objects (i.e. #68).

## Related

- [FP-1878](https://jira.tacc.utexas.edu/browse/FP-1878)
- fixes https://github.com/TACC/Core-Styles/pull/68
- required by https://github.com/TACC/Core-CMS/pull/569

## Changes

- rename `o-table-wrap--overflow-scroll.extra` → `o-table-wrap--overflow-scroll-extra`

## Testing

1. `npm run build`
2. `npx serve demo`
3. Confirm http://localhost:3000/components/detail/o-table-wrap--overflow-scroll-extra.html loads.
4. Confirm O Table Wrap > Overflow Scroll (Extra) link in nav loads same page.

## UI

### [Locally](http://localhost:3000/components/detail/o-table-wrap--overflow-scroll-extra.html)

![fix fp-1878 overflow-scroll-extra](https://user-images.githubusercontent.com/62723358/202238747-fe4d7d0b-8e24-4046-982b-508537be3800.png)

### [Remotely](https://github.com/TACC/Core-Styles/pull/69)

![remote tup-cms oversflow scrollextra](https://user-images.githubusercontent.com/62723358/202276566-170f9084-af59-4099-aa2e-8213d45487ea.png)
